bitnami/mongodb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 7.10.12
+version: 7.10.13
```

Comment:
It would be a minor change, please update the version properly.

Reply:
bumped to 7.11.0